--- /tmp/venv-pyrdp-original/lib/python3.13/site-packages/pyrdp/mitm/FileMapping.py	2026-01-05 10:02:44.365723816 +0000
+++ /opt/jumphost/venv/lib/python3.13/site-packages/pyrdp/mitm/FileMapping.py	2026-01-02 12:40:18.981353209 +0000
@@ -9,7 +9,7 @@
 import tempfile
 from logging import LoggerAdapter
 from pathlib import Path
-from typing import io
+import typing
 
 
 class FileMapping:
@@ -18,7 +18,7 @@
     transferred over RDP.
     """
 
-    def __init__(self, file: io.BinaryIO, dataPath: Path, filesystemPath: Path, filesystemDir: Path, log: LoggerAdapter):
+    def __init__(self, file: typing.BinaryIO, dataPath: Path, filesystemPath: Path, filesystemDir: Path, log: LoggerAdapter):
         """
         :param file: the file handle for dataPath
         :param dataPath: path where the file is actually saved
